bench/mimalloc/optimized/heap.ll

```diff
@@ -877,9 +879,9 @@ define hidden zeroext i1 @mi_heap_contains_block(ptr noundef readnone captures(a
   %20 = getelementptr inbounds nuw i8, ptr %10, i64 144
   %21 = load i64, ptr %20, align 16, !tbaa !61
   %22 = lshr i64 %19, %21
-  %.idx.i = mul nuw nsw i64 %22, 80
-  %23 = getelementptr i8, ptr %12, i64 216
-  %24 = getelementptr i8, ptr %23, i64 %.idx.i
+  %23 = getelementptr inbounds nuw i8, ptr %12, i64 160
+  %23 = getelementptr inbounds nuw [1 x %struct.mi_page_s], ptr %23, i64 0, i64 %22
+  %24 = getelementptr inbounds nuw i8, ptr %23, i64 56
```

Comment:
Missing fold to combine constant GEP indices: https://llvm.godbolt.org/z/xonqM9Wqj